content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
+With LXD, you can create both system containers and virtual machines. You should use a system container to leverage the smaller size and increased performance if all functionality you require is compatible with the kernel of your host operating system. If you need a fully isolated environment or functionality that is not supported by the OS kernel of your host system, use a virtual machine.
+
+![Virtual machines and system containers](/static/img/lxd/virtual-machines-vs-system-containers.svg "Virtual machines and system containers")
+
+# Features
+Some of the biggest features of LXD are:
+
+Core API
+: * [Secure by design](https://linuxcontainers.org/lxd/docs/master/security) (through unprivileged containers, resource restrictions, authentication, ...)
+  * [Intuitive](https://linuxcontainers.org/lxd/docs/master/rest-api) (with a simple, clear API and crisp command line experience)
+  * [Scalable](https://linuxcontainers.org/lxd/docs/master/clustering) (from containers on your laptop to clusters of thousands of compute nodes)
+  * [Event based](https://linuxcontainers.org/lxd/docs/master/events) (providing logging, operation, and lifecycle events)
+  * [Remote usage](https://linuxcontainers.org/lxd/docs/master/remotes) (same API used for local and network access)
+  * [Project support](https://linuxcontainers.org/lxd/docs/master/projects) (as a way to compartmentalize sets of images and profiles)
+
+Instances and profiles
+: * [Image based](https://images.linuxcontainers.org) (with images for a wide variety of Linux distributions, published daily)
+  * [Containers](https://linuxcontainers.org/lxd/docs/master/containers) (the most complete implementation of LXD instances)
+  * [Virtual machines](https://linuxcontainers.org/lxd/docs/master/virtual-machines) (implemented through the use of qemu)
+  * [Configurable through profiles](https://linuxcontainers.org/lxd/docs/master/profiles) (applicable to both containers and virtual machines)
+
+Backup and export
+: * [Backup and recovery](https://linuxcontainers.org/lxd/docs/master/backup) (for all objects managed by LXD)
+  * [Snapshots](https://linuxcontainers.org/lxd/docs/master/instances#snapshot-scheduling) (to save and restore the state of an instance)
+  * [Container and image transfer](https://linuxcontainers.org/lxd/docs/master/image-handling) (between different hosts, using images)
+  * [Live migration](https://linuxcontainers.org/lxd/docs/master/migration) (using CRIU)
+
+Configurability
+: * [Multiple storage backends](https://linuxcontainers.org/lxd/docs/master/storage) (with configurable storage pools and storage volumes)
+  * [Network management](https://linuxcontainers.org/lxd/docs/master/networks) (including bridge creation and configuration, cross-host tunnels, ...)
+  * [Advanced resource control](https://linuxcontainers.org/lxd/docs/master/instances#resource-limits-via-limitskernellimit-name) (CPU, memory, network I/O, block I/O, disk usage and kernel resources)
```

Comment:
The link does not guide you to the expected header.
I assume it can be a problem with a other links too.
I guess it is related to #446.

Reply:
Indeed, for some reason, it's `https://linuxcontainers.org/lxd/docs/master/instances#resource-limits-via-`

Reply:
Guess it cannot handle special characters. Link fixed.